app/controllers/concerns/sar_internal_review_cases_params.rb

```diff
@@ -24,4 +24,38 @@ def create_sar_internal_review_params
     )
   end
 
+  def process_sar_closure_params
+    params.require(:sar_internal_review).permit(
+      :date_responded_dd,
+      :date_responded_mm,
+      :date_responded_yyyy,
+      :late_team_id,
+    ).merge(refusal_reason_abbreviation: missing_info_to_tmm)
+  end
```

Comment:
Not sure about this one,  do you actually use it?

Reply:
Thanks I will take a look at remove if not :)